src/EventListener/CheckCarHistoryEventHandler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace App\EventListener;
+
+use App\Event\CheckCarHistoryEvent;
+use App\Repository\CarRepository;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+readonly class CheckCarHistoryEventHandler implements MessageHandlerInterface
+{
+
+    public function __construct(
+        private  CarRepository $carRepository,
+        private  ParameterBagInterface $parameterBag, 
```

Comment:
don't use paarmeter bag, inject in services yaml env variable as we did in the past